packages/webdriverio/src/commands/element/getProperty.js

```diff
@@ -3,16 +3,16 @@
  *
  * <example>
     :getProperty.js
-    it('should demonstrate the getCSSProperty command', () => {
+    it('should demonstrate the getProperty command', () => {
         var elem = $('body')
-        var color = elem.getProperty('tagName')
-        console.log(color) // outputs: "BODY"
+        var tag = elem.getProperty('tagName')
+        console.log(tag) // outputs: "BODY"
     })
  * </example>
  *
  * @alias element.getProperty
  * @param {String} property  name of the element property
- * @return {Object|String} the value of the property of the selected element
+ * @return {Object|String|Boolean|Number|null} the value of the property of the selected element
```

Comment:
Do you know why null is ignored in types?

Reply:
I'm not sure. It seems to do the same thing with `getAttribute`.